js/autotrack/reactNavigation.js

```diff
@@ -84,6 +85,10 @@ export const withReactNavigationAutotrack = track => AppContainer => {
     }
   }
 
+  HeapNavigationWrapper.displayName = `withReactNavigationAutotrack(${getComponentDisplayName(
```

Comment:
How married are we to `withReactNavigationAutotrack`?  For one thing, the word isn't right (should be autocapture), but for another it doesn't _read_ properly in English when you see it in a hierarchy!  You have to invert the order of the tokens in your head to make any sense of it.  What if we had this displayName be something like `HeapAutocaptured`, so that when you read the hierarchy it says `HeapAutocaptured(Main Menu)` or somesuch.

Reply:
This seems to be a relatively common naming convention for HOCs (e.g. `withRouter`, `withNavigation`, etc.), so I'd prefer we just stick with this